src/app/Component/chatroom/ChatRoom.js

```diff
@@ -70,29 +72,30 @@ class ChatRoom extends Component {
   }
 
   send = (index) => {
+    let tempmsg;
+    tempmsg=this.message.current.value.trim();
```

Comment:
why are we declaring and assigning in two separate lines? can't we do it in a single line?